api-reference/10 UI Widgets/dxFileUploader/1 Configuration/dialogTrigger.md

```diff
@@ -5,7 +5,38 @@ default: undefined
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies the HTML element a click on which invokes file upload dialog.
```

Comment:
```suggestion
Specifies the HTML element which invokes the file upload dialog.
```

Reply:
The element cannot invoke the dialog, only an action (click or hover) can do it.

Reply:
I do not agree. Yes, you have to click it, but it is this element that invokes to dialog. If you clicked somewhere else, it would not invoke the dialog.

Reply:
ok